### PR TITLE
[FUSEQE-6028] Remove external services from support page

### PR DESCRIPTION
**Pull request by Stefan365**
//test: `@support-page-download-specific-diagnostic`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Support page | Export diagnostic of single integration  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Support page | Export diagnostic of single integration  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/resources/features/other/support-page.feature [hunk 1]
```diff
@@ -38,59 +36,45 @@ Feature: Support page
...
-    Then check that position of connection to fill is "Finish"
-
-    # select salesforce connection
-    When select the "QE Salesforce" connection
-    And select "Create or update record" integration action
-    And select "Contact" from "sObjectName" dropdown
+    When select the "PostgresDB" connection
+    And select "Periodic SQL invocation" integration action
+    Then check "Next" button is "Disabled"
+    Then fill in periodic query input with "SELECT * FROM CONTACT" value
```
Comment (mkralik3) [inline ui-tests/src/test/resources/features/other/support-page.feature]: `When` is better for steps without validation.

File: ui-tests/src/test/resources/features/other/support-page.feature [hunk 2]
```diff
@@ -38,59 +36,45 @@ Feature: Support page
...
-      | user.name       | FirstName; LastName  |
+    Then check visibility of page "Choose a Finish Connection"
+    When select the "Log" connection
+    And fill in values by element data-testid
+      | contextloggingenabled | true |
+      | bodyloggingenabled    | true |
 
-    And scroll "top" "right"
-    And click on the "Done" button
+    Then click on the "Done" button
```
Comment (mkralik3) [inline ui-tests/src/test/resources/features/other/support-page.feature]: Same here (`And`)

File: ui-tests/src/test/resources/features/other/support-page.feature [hunk 3]
```diff
@@ -38,59 +36,45 @@ Feature: Support page
...
 
-    # finish and save integration
     When click on the "Save" link
-    And set integration name "my-integration"
+    And set integration name "support-page-integration"
     And publish integration
-    Then wait until integration "my-integration" gets into "Running" state
+    Then Integration "support-page-integration" is present in integrations list
+    And wait until integration "support-page-integration" gets into "Running" state
```
Comment (mkralik3) [inline ui-tests/src/test/resources/features/other/support-page.feature]: Same here (`And` is related to Then, but this is not validation step)

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Support page | Export diagnostic of single integration  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Support page | Export diagnostic of single integration  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**